src/SdkCommon/ClientRuntime/ClientRuntime/RetryDelegatingHandler.cs

```diff
@@ -39,7 +35,7 @@ public RetryDelegatingHandler() : base()
         /// the default retry policy base on Exponential Backoff.
         /// </summary>
         /// <param name="innerHandler">Inner http handler.</param>
-        public RetryDelegatingHandler(DelegatingHandler innerHandler)
+        public RetryDelegatingHandler(HttpMessageHandler innerHandler)
```

Comment:
This is not a breaking change.  DelegatingHandler derives from HttpMessageHandler, so this makes the these types more flexible.

The reason I changed this is so I could use the "FakeHttpHandler" in testing these handlers.

Reply:
This is a binary breaking change. You can not drop in a new compiled binary with existing binaries with this change. 
If you recompiled the code that used the library, yes, this would compile and function, but it is most definitely breaking change, and cannot be transparently rolled out on top of existing binaries.
​
Attempting to do so will produce:
`Unhandled Exception: System.MissingMethodException: Method not found: 'Void RetryDelegatingHandler..ctor(System.Net.Http.DelegatingHandler)'. `

Reply:
Changed this to use a new constructor overload to avoid the binary breaking change.